extensions/wikia/AdEngine/js/spec/lookup/prebid/adaptersRegistry.spec.js

```diff
@@ -0,0 +1,110 @@
+/*global describe, it, modules, expect, spyOn, beforeEach*/
+/*jshint maxlen:200*/
+describe('adaptersRegistry', function () {
+	'use strict';
+
+    function AdapterMock(name, enabled) {
+        this.getName = function getName() {
+            return name;
+        };
+
+        this.isEnabled = function isEnabled() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make this function anonymous by removing its name: 'function() {...}'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3ANamedFunctionExpression)